tfhe/src/integer/parameters/mod.rs

```diff
@@ -117,3 +119,44 @@ pub const PARAM_MESSAGE_1_CARRY_1_KS_PBS_32_BITS: WopbsParameters = WopbsParamet
     ciphertext_modulus: CiphertextModulus::new_native(),
     encryption_key_choice: EncryptionKeyChoice::Big,
 };
+
+pub struct CiphertextConformanceParams {
+    pub shortint_params: shortint::parameters::CiphertextConformanceParams,
+    pub num_blocks_per_integer: usize,
+}
```

Comment:
Down the line those are radix conformance params, but we will want to have the distinction vs. CRT, if you can have an issue about it and put those in the radix ciphertext module ?

Reply:
Sure, I'll move it
What would the issue contain?

Reply:
Indicate that Crt Ciphertexts are different compared to Radix Ciphertexts and will required their own handling down the line

Reply:
There's no `ciphertext::radix` module
I'll rename them to `Radix___`

Reply:
Renamed